### DISCUSSION TITLE
Problem upgrading next 9.1.4 to the latest (using custom server...)

### DISCUSSION THREAD

**Discussion by omribenari** [vercel/next.js#17650] [Help]
Dos anyone had such issues ?

**Comment by mxmtsk** [comment]
> Dos anyone had such issues ?

What kind of issues? In general no I did not have any issues upgrading.

Did you check Release Notes to see if any of the changes affected your project?
You can find them here: https://nextjs.org/blog

Otherwise you should provide more context like error messages you face.
